### PR TITLE
Update paml 4.9e

### PR DESCRIPTION
**Pull request by romainstuder**
Update paml 4.9c to 4.9e

### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [x] Removed the `revision` line, if any, when bumping the version number?
- [x] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

**Comment by iMichka** [comment]
Hi. Please don't create new branches each time.
You can `git commit --amend` your current commit, and the `git push --force <yourfork> <branch-name>`.

**Comment by jonchang** [comment]
```
homebrew/science/paml:
  * Use the `chmod` Ruby method instead of `system "chmod `
```

File: paml.rb [hunk 1]
```diff
@@ -17,9 +17,9 @@ class Paml < Formula
 
   def install
     # Change some permissions that were wrong in the tarball archives"
-    chmod "u=wr", "examples/DatingSoftBound/*"
-    chmod "u=wr", "bin/*"
-    chmod "u=wr", "docs/*"
```
Comment (jonchang) [inline paml.rb]: Why not just `Dir["*"]`?

File: paml.rb [hunk 2]
```diff
@@ -16,6 +16,13 @@ class Paml < Formula
   end
 
   def install
+    # Change some permissions that were wrong in the tarball archives"
+    chmod "u=wr", "examples/DatingSoftBound/SeedUsed"
+    chmod "u=wr", "examples/DatingSoftBound/mtCDNApri.trees"
+    chmod "u=wr", "examples/DatingSoftBound/out.txt"
+    chmod "u=wr", "doc/pamlHistory.txt"
+    chmod "u=wr", Dir["src/*"]
```
Comment (jonchang) [inline paml.rb]: `chmod "u+rwX", "."`

**Comment by jonchang** [comment]
@BrewTestBot test this please

**Comment by jonchang** [comment]
Thanks @romainstuder, this has been merged. Note that our commit style is `paml 4.9e`, see [this document](http://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request.html) for details.

**Pull request closed without merging**